app/_hub/salt/salt/_metadata.yml

```diff
@@ -11,10 +11,16 @@ desc: Integrate Kong API Gateway with Salt Security Discovery & Prevention for A
 
 enterprise: true
 techpartner: true
+konnect: true
 
 kong_version_compatibility:
   community_edition:
     compatible:
+      - 3.2.x
+      - 3.1.x
+      - 3.0.x
+      - 2.9.x
+      - 2.8.x
```

Comment:
Did you mean to update just the `community_edition` section, or did you want to list compatibility with Kong Enterprise? If it's the latter, you'll also need to list these same versions after line 31, in the `enterprise_edition` section.

Reply:
that is correct, these versions need to shift to in the `enterprise edition`

Reply:
added versions to Enterprise section